segmentstore/server/src/test/java/io/pravega/segmentstore/server/logs/operations/StreamSegmentTruncateOperationTests.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
```

Comment:
Please revert the changes to license header.

Reply:
fixed